### PR TITLE
Exercises readme

### PR DESCRIPTION
**Pull request by shhavel**
## Description

Add Exercises description.

## Additional changes

- [ ] The first task has changed a bit. As I understand func `cube` cannot be obtained from [Math.pow(base, exponent)](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/pow) using [bind()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_objects/Function/bind) - the first arg can be binded - so the base will be fixed then, not the power.
- [ ] YouTube video link with playlist identifier.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
 # Partial function application and Currying
 
-[![Композиция, каррирование, частичное применение](https://img.youtube.com/vi/ND8KQ5xjk7o/0.jpg)](https://www.youtube.com/watch?v=ND8KQ5xjk7o)
+[![Композиция, каррирование, частичное применение](https://img.youtube.com/vi/ND8KQ5xjk7o/0.jpg)](https://www.youtube.com/watch?v=ND8KQ5xjk7o&list=PLHhi8ymDMrQZad6JDh6HRzY1Wz5WB34w0)
```
Comment (tshemsedinov) [inline README.md:3]: We use links without playlists everywhere

File: Exercises/1-power.test [hunk 2]
```diff
@@ -25,14 +25,22 @@
   name: 'square',
   length: [10, 25],
   cases: [
+    [5, 25],
+    [6, 36],
+    [9, 81],
   ]
 }, {
-  name: 'cube',
+  name: 'powerOfThree',
```
Comment (tshemsedinov) [inline Exercises/1-power.test:33]: Why?

File: Exercises.en.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+## [5. Curry](Exercises/5-curry.js)
+
+Given function `someFunc()` which accepts `n` arguments (e.g. 3 arguments).
+Implement function `curry()` that generates other function which accepts `someFunc()` arguments partially and supplies them to `someFunc()`:
+
+```js
+const func = curry(3, someFunc);
+
+func(a)(b)(c) == someFunc(a, b, c) //=> true
```
Comment (tshemsedinov) [inline Exercises.en.md:31]:
```suggestion
func(a)(b)(c) === someFunc(a, b, c) //=> true
```

File: Exercises.en.md [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+
+func(a, b)(c) == someFunc(a, b, c) //=> true
```
Comment (tshemsedinov) [inline Exercises.en.md:33]:
```suggestion
func(a, b)(c) === someFunc(a, b, c) //=> true
```

File: Exercises.en.md [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+
+func(a)(b, c) == someFunc(a, b, c) //=> true
```
Comment (tshemsedinov) [inline Exercises.en.md:35]:
```suggestion
func(a)(b, c) === someFunc(a, b, c) //=> true
```

File: Exercises.en.md [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+Use a function inside a new function.
+
+## [4. Bind](Exercises/4-bind.js)
+
+Use method bind() to existing function to apply preceding arguments and obtain a new function.
+
+## [5. Curry](Exercises/5-curry.js)
+
+Given function `someFunc()` which accepts `n` arguments (e.g. 3 arguments).
+Implement function `curry()` that generates other function which accepts `someFunc()` arguments partially and supplies them to `someFunc()`:
```
Comment (tshemsedinov) [inline Exercises.en.md:26]: Line max length is 80, only URLs can be longer

**Comment by tshemsedinov** [comment]
Partially landed in https://github.com/HowProgrammingWorks/PartialApplication/commit/28b77de97cfc24422e06da219ce28c3629bea07c

File: Exercises/1-power.test (hunk 2 above)
Comment (shhavel) [reply Exercises/1-power.test:33]: @tshemsedinov Maybe I got it wrong. I think function `cube` cannot be obtained from [Math.pow(base, exponent)](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/pow) using [bind()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_objects/Function/bind) - the first argument can be binded - so the base will be fixed, not the exponent (number 3 for cube). Thank you.

Comment (shhavel) [reply Exercises/1-power.test:33]: Also [expectation for function `cube`](https://github.com/HowProgrammingWorks/PartialApplication/blob/master/Exercises/1-power.test#L42-L44) don't match to it's name.

**Pull request closed without merging**